### PR TITLE
add cereal@1.3.2

### PR DESCRIPTION
**Pull request by tongke6**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (cereal) have been updated in this PR. Please review the changes.

**Comment by tongke6** [comment]
@bazel-io skip_check unstable_url

**Comment by tongke6** [comment]
@bazel-io skip_check unstable_url

File: modules/cereal/1.3.2/overlay/BUILD [hunk 1]
```diff
@@ -0,0 +1,20 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all",
+    srcs = glob(["**"]),
+)
+
+cmake(
```
Comment (meteorcloudy) [inline modules/cereal/1.3.2/overlay/BUILD:10]: nit: it would be nicer to actually include a native BUILD file for this.

**Review by meteorcloudy — COMMENTED** [review]
Looks like it's broken with Bazel 6 and windows.

File: modules/cereal/1.3.2/overlay/BUILD (hunk 1 above)
Comment (tongke6) [reply modules/cereal/1.3.2/overlay/BUILD]: done, use native cc_library.

File: modules/cereal/1.3.2/source.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/USCiLab/cereal/archive/refs/tags/v1.3.2.tar.gz",
+    "integrity": "sha256-FqetmzG6WIDaxV1itdbyQ8PryNRqNRQUnla15+qB+F8=",
+    "strip_prefix": "cereal-1.3.2",
+    "overlay": {
```
Comment (meteorcloudy) [inline modules/cereal/1.3.2/source.json:5]: overlay is only supported in Bazel 7, so please remove "6.x" in presubmit.yml.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: modules/cereal/1.3.2/source.json (hunk 2 above)
Comment (tongke6) [reply modules/cereal/1.3.2/source.json:5]: Got it, thanks!

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**